elenaps/tips&triks/el-color/readme.md

```diff
@@ -0,0 +1,98 @@
+![Tips&Triks](./images/PortadaTips&Triks.jpg)
+
+## [Tips&Triks] Diseño para desarrolladores - El Color
+
+Mis compañeros de **No Country for Geeks** llevan tiempo pidiéndome que escriba tips and triks para desarrolladores que desean diseñar sus propias cabeceras de sus blogs personales.
+
+El requisito básico es tener la predisposición de aprender algo que no parece muy extraño (¿quién no ha combinado una camiseta con un pantalón o dibujado y coloreado algo alguna vez?) pero que tiene su intríngulis si se quiere hacer bien.
+
+Espero que con los siguientes tips puedas dar algún paso más en este mundillo, y como no podría ser de otra manera, lo primero que hay que hacer es investigar y curiosear.
+
+## INVESTIGA Y CURIOSEA
+Vamos a ver cuál es la mejor forma de transmitir tu mensaje a través de un elemento ilustrativo.
+
+Escribe en una hoja palabras que creas que pueden representar tu post, tienes que conseguir abstraer el concepto principal preguntándote: 
+¿Cómo puedes invitar a alguien a que lea tu post únicamente con una imagen?, ¿qué quieres destacar?, ¿qué elemento podría representarlo?,  ¿quién va a leerlo? o ¿cómo lo hacen en otros blogs o páginas tecnológicas que sueles usar de referencia?
```

Comment:
Take care with break lines. Markdown only support double break lines. Another options are going to be concatenated.

Reply:
+1